packages/client/types/constEnums.ts

```diff
@@ -69,6 +66,11 @@ export const enum DragAttribute {
   DROPZONE = 'data-dropzone'
 }
 
+export const enum DrawerTypes {
```

Comment:
since we've already defined this as `TeamDrawerEnum` on the server, we should use the server definition as the single source of truth.

on the client we can access it like  `import {TeamDrawerEnum} from '../__generated__/ToggleTeamDrawerMutation.graphql`

for context, i think it generally makes sense to opt for string unions over enums if the values of the string unions are human readable (eg they are names like "agenda" and aren't color hex codes)